src/main/java/mate/academy/rickandmorty/repository/Repository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.rickandmorty.repository;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.Character;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface Repository extends JpaRepository<Character, Long> {
```

Comment:
Repository is bad name. Too generic. 
```suggestion
public interface Repository extends JpaRepository<Character, Long> {
```